### PR TITLE
Update ui_design.md

### PR DESCRIPTION
**Pull request by BVrolyk**
7 - Changed "a define set" to "a defined set".

### CODE REVIEW

File: docs/ui_design.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This document outlines a proposed architecture for a Helm UI MVP.
 
 Architectural design goals:
 
-- Outline a definite set of feature requirements that will ship with the MVP.
+- Outline a definited set of feature requirements that will ship with the MVP.
```
Comment (sgoings) [inline docs/ui_design.md]: Not sure you wanted 'definited'.

File: docs/ui_design.md [hunk 2]
```diff
@@ -40,4 +40,4 @@ As the above requires no API interaction with a backend service, it is reasonabl
 
 #### Cons
 - not the easiest bootstrapping approach, requires thinking about future CRUD features and spending some time to choose the right tool for that future job
-- requires engineering TLC over time: it's a dev project, not a CMS instance
\ No newline at end of file
+- requires engineering TLC over time: it's a dev project, not a CMS instance
```
Comment (sgoings) [inline docs/ui_design.md]: what changed on this line?

File: docs/ui_design.md (hunk 1 above)
Comment (BVrolyk) [inline docs/ui_design.md]: Nope looking for "defined" and torturing you all as I learn the tools.

File: docs/ui_design.md (hunk 2 above)
Comment (BVrolyk) [inline docs/ui_design.md]: No idea, I believe I clicked on it.

Comment (sgoings) [inline docs/ui_design.md]: That'll teach you to click on things. :wink:

**Pull request closed without merging**